package-lock.json

```diff
@@ -13586,7 +13586,7 @@
     },
     "pify": {
       "version": "2.3.0",
-      "resolved": "https://registry.npmjs.org/pify/-/pify-2.3.0.tgz",
+      "resolved": "http://registry.npmjs.org/pify/-/pify-2.3.0.tgz",
```

Comment:
why this was changed to http?

Reply:
It's weird. This change was introduced by `npm install`.